data/core/tips.json

```diff
@@ -0,0 +1,176 @@
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Afraid of wild life? Yell untill it goes away.  But beware of what comes instead." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Animals and zombies are not friends.  You can take advantage of that." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Door is not the only way in.  Or out if that matters." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Don't fight against the odds.  Running is no shame in survival." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "You don't have to sit next to fire to use it." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Many items have special actions.  Try to find out what they do." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Dangerous place is not always equal to good loot.  Use your head." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Getting in is not a problem.  Think of way out and don't get cornered." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "There are often more ways then one to achieve something.  Use your head." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Survival order: shelter, water, food, everything else." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Balance armor versus encumbrance.  Even dropping a backpack can help in melee." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Scared of melee? Throw some things at enemies or use reach weapons like spears." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Makeshift weapons can break easily, but they are weapons, and can be made again." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Not much survives a car at 120mph.  Can you?" ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "You don't want an overgrown cockroach in your rotten food.  Get rid of it." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Medicine can help you survive in a pinch but avoid overdose and addictions." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Use your map wisely. Prioritize looting places that will get yout what you need." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Wounds heal over time.  Bandaging and disinfecting can speed up the process." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Don't get grabbed by the zombie.  They can bite you and infect the wound." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Feeling odd after waking up? Your health might not be well." ]
```

Comment:
Since the concept of "health" isn't obviously exposed to players, we should give some tips here on how to improve it. E.g

> Feeling odd after waking up? Try eating healthier and the situation may improve.